src/components/common/PageHeader/styles.module.css

```diff
@@ -7,7 +7,7 @@
   border-bottom: 1px solid var(--color-border-light);
   background-color: var(--color-background-paper);
   z-index: 1;
-  width: 100%;
+  width: calc(100% - var(--space-6));
```

Comment:
My bad! I've added box-sizing to the container and forgot about the horizontal padding.

Reply:
I tried it without the box-sizing but then the 0.5px padding we add doesn't work anymore so we would have to find another way.